app/static/css/themes/nord-search.css

```
@@ -0,0 +1,51 @@
+.autocomplete {
+    position: relative;
+    display: inline-block;
+    width: 100%;
+}
+
+.autocomplete-items {
+    position: absolute;
+    border: 1px solid #685e79;
+    border-bottom: none;
+    border-top: none;
+    z-index: 99;
+
+    /*position the autocomplete items to be the same width as the container:*/
+    top: 100%;
+    left: 0;
+    right: 0;
+}
+
+.autocomplete-items div {
+    padding: 10px;
+    cursor: pointer;
+    color: #3B4252 !important;
+    background-color: #ECEFF4 !important;
```

These properties probably shouldn't be flagged as `!important`, since that overrides the properties in `.autocomplete-active`. 

```suggestion
    color: #3B4252;
    background-color: #ECEFF4;
```